testsuite/Makefile

```diff
@@ -17,46 +17,43 @@ NO_PRINT=`$(MAKE) empty --no-print-directory >/dev/null 2>&1 \
 FIND=find
 include ../config/Makefile
 
+# Maximum number of retries for directories with failing tests
+# Default = 1 for native Windows, 0 for all other platforms
+ifeq ($(filter msvc mingw,$(TOOLCHAIN)),)
+MAX_RETRIES=0
+else
+MAX_RETRIES=1
+endif
+
```

Comment:
Oops - at the very least, it should have been above the `include`, to allow `config/Makefile` to override the automatic default! I'll move it to the Windows Makefiles and update `configure` appropriately (if it's going "globally", I guess MAX_TESTSUITE_DIR_RETRIES might be a better name?)